portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -38,12 +40,18 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Can you add comments indicating what all the  servlet handlers do (what their purpose is, what they're expected to return)?

Reply:
yes